src/Ocelot/LoadBalancer/LoadBalancers/InMemoryStickySessionStorage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Concurrent;
+
+namespace Ocelot.LoadBalancer.LoadBalancers;
```

Comment:
It must be moved up one level because it is not balancer actually!

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/1405/commits/a2d1e3be39a80eec6a025797662414d6e389b198 ✔️